bitnami/wordpress/Chart.yaml

```diff
@@ -31,4 +31,4 @@ name: wordpress
 sources:
   - https://github.com/bitnami/bitnami-docker-wordpress
   - http://www.wordpress.com/
-version: 10.4.7
+version: 10.4.8
```

Comment:
I would bump the minor since you are adding new functionality. Even if the configmap was created before it was not being added to the deployment. so IMHO this PR is the one adding a new feature